sample_apps/rust/examples/cleanup_sample.rs

```diff
@@ -0,0 +1,113 @@
+pub mod utils;
+use crate::utils::timestream_helper;
+use clap::Parser;
+use std::error::Error;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    let args = timestream_helper::Args::parse();
+
+    let client = timestream_helper::get_connection(&args.region)
+        .await
+        .expect("Failed to get connection to Timestream");
+
+    match client
+        .describe_table()
+        .database_name(&args.database_name)
+        .table_name(&args.table_name)
+        .send()
+        .await
+    {
+        Ok(describe_table_output) => {
+            if let Some(table) = describe_table_output.table() {
+                if let Some(magnetic_store) = table.magnetic_store_write_properties() {
```

Comment:
What would happen here if `table.magnetic_store_write_properties()` returned `None`?

Reply:
Then we exit out if the `if let` chain and follow on to line 25 to delete the table. `if let` is like match but only cares about one path.